nautilus_trader/live/execution_engine.py

```diff
@@ -1842,43 +1843,37 @@ def _reconcile_execution_mass_status(
                     f"not in `reconciliation_instrument_ids` include list",
                 )
                 continue
+            reconciliation_instruments.append(self._cache.instrument(instrument_id))
 
-            self._log.info(
-                f"Attempting to adjust fills for {instrument_id}",
-                LogColor.BLUE,
-            )
-            instrument = self._cache.instrument(instrument_id)
-            if instrument:
-                self._log.info(
-                    f"Calling adjust_fills_for_partial_window for {instrument_id}",
-                    LogColor.BLUE,
-                )
-                adjusted_orders_for_instrument, adjusted_fills_for_instrument = (
-                    adjust_fills_for_partial_window(
-                        mass_status,
-                        instrument,
-                        self._log,
-                    )
-                )
-
-                # Remove old orders and fills for this instrument
-                for venue_order_id in list(final_orders.keys()):
-                    order = final_orders[venue_order_id]
-                    if order.instrument_id == instrument_id:
-                        del final_orders[venue_order_id]
-
-                for venue_order_id in list(final_fills.keys()):
-                    fills = final_fills[venue_order_id]
-                    if fills and fills[0].instrument_id == instrument_id:
-                        del final_fills[venue_order_id]
+        self._log.debug(
```

Comment:
I think we should retain the original log levels here and add a config to reduce them in a follow-up.